### PR TITLE
DOC: update README.md with new features of v1.0.0a1.

### PR DESCRIPTION
**Pull request by phmbressan**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Code base additions (bugfix, features)
- [ ] Code maintenance (refactoring, formatting, renaming, tests)
- [X] ReadMe, Docs and GitHub maintenance
- [ ] Other (please describe):

## Pull request checklist

Please check if your PR fulfills the following requirements, depending on the type of PR:

- ReadMe, Docs and GitHub maintenance:

  - [X] Spelling has been verified
  - [ ] Code docs are working correctly 

## What is the current behavior?

Documentation was out of date regarding features of `v1.0.0a1`.

## What is the new behavior?

Readme.md was updated with essential info regarding `HybridMotor` and `LiquidMotor`.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -318,6 +334,12 @@ Here is just a quick taste of what RocketPy is able to calculate. There are hund
 
 ![6-DOF Trajectory Plot](https://raw.githubusercontent.com/RocketPy-Team/RocketPy/master/docs/static/rocketpy_example_trajectory.svg)
 
+If you want to see the trajectory on Google Earth, RocketPy acn easily export a KML file for you:
+
+```python
+test_flight.export_kml(file_name="test_flight.kml")
```
Comment (Gui-FernandesBR) [inline README.md:340]: I will give a candy to whoever adds a good picture of trajectory plotted on Google Earth

extra candy if it is a GIF

File: README.md [hunk 2]
```diff
@@ -175,6 +176,12 @@ A typical workflow starts with importing these classes from RocketPy:
 from rocketpy import Environment, Rocket, SolidMotor, Flight
 ```
 
+An optional step is to import datetime, which is used to define the date of the simulation:
```
Comment (Gui-FernandesBR) [inline README.md:179]: The diagram of classes needs to be refactored before v1, don't let me forget it please

**Review by Gui-FernandesBR — APPROVED** [review]
Well done, @phmbressan !!

I updated a few things in the file. Please check it and merge whenever you want.
Could you please confirm that the simulation steps (given in the README) work on your machine?

Ty again!

**Comment by phmbressan** [comment]
The README.md simulations are working correctly here, I will merge this PR, but we ought to remember to update the diagrams in `v1.0.0`.

**Pull request merged**